addons/road-generator/nodes/road_segment.gd

```diff
@@ -188,6 +188,44 @@ func check_rebuild() -> bool:
 	return false
 
 
+func generate_edge_curves():
+	if not is_instance_valid(container):
+		return
+	if not is_instance_valid(start_point) or not is_instance_valid(end_point):
+		return
+	if not container.create_edge_curves:
+		clear_edge_curves()
+		return
+
+	# Find the road edge positions
+	var matched_lanes = self._match_lanes()
+	if len(matched_lanes) == 0:
+		return
+
+	var _par = get_parent()
+
+	# Add edge curves
+	var edge_R: Path = _par.get_node_or_null("edge_R")
+	var edge_F: Path = _par.get_node_or_null("edge_F")
+	var temp_offset = 14
```

Comment:
I gather this is the main next step, to make offset account for lane count * lane width + shoulder width?